content/changes.md

```diff
@@ -8,6 +8,22 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
+## June 30, 2024
+
+### Bug Fixes 
+
+- **Demo Data - Scoring Inaccurate** 
+
+  We disabled the option to generate demo data on new teams in order to keep our demo data consistent for all our customers. 
+ 
```

Comment:
```suggestion
```

This isn't fixed yet, so it's too soon to alert customers of our rather embarrassing temporary workaround. Once we do fix this, though, this will be a good thing to include.